### PR TITLE
update: enhance Clean task

### PR DESCRIPTION
**Pull request by mrostamii**
**Description**
- services must be stopped!
- if the services file wouldn't be removed, would conflict if the user will try to setup the network again.

### CODE REVIEW

File: playbooks/clean.yml [hunk 1]
```diff
@@ -31,3 +31,25 @@
...
+    - name: Clean services file
+      file:
+        path: "/etc/systemd/system/{{ item }}"
+        state: absent
+      with_items:
+        - bor.service
+        - heimdalld.service
+        - heimdalld-bridge.service
+        - heimdalld-rest-server.service
+      become: yes
```
Comment (vcastellm) [inline playbooks/clean.yml:55]: Missing New line here

**Review by vcastellm — CHANGES_REQUESTED** [review]
LGTM aside from the comment

File: playbooks/clean.yml (hunk 1 above)
Comment (mrostamii) [reply playbooks/clean.yml:55]: @vcastellm Which line do you mean?

Comment (vcastellm) [reply playbooks/clean.yml:55]:
```suggestion
      become: yes

```

Comment (vcastellm) [reply playbooks/clean.yml:55]: Empty new line at the end of the file, GH is showing the red sign

**Review by vcastellm — APPROVED** [review]

**Pull request merged**